lib/linguist/generated.rb

```diff
@@ -54,6 +54,7 @@ def generated?
       xcode_file? ||
       cocoapods? ||
       carthage_build? ||
+      generated_graphql_relay_js? ||
```

Comment:
Maybe remove `_js` at the end of the name, now that we're marking the whole directory as generated?

Reply:
done!